syncmaster/scheduler/settings/__init__.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+
+from syncmaster.settings import SyncmasterSettings
+
+
+class SchedulerSettings(SyncmasterSettings):
+    """Celery scheduler settings.
+
+    Examples
+    --------
+
+    .. code-block:: bash
+
+        SYNCMASTER__SCHEDULER__TRANSFER_FETCHING_TIMEOUT=200
+    """
+
+    TRANSFER_FETCHING_TIMEOUT: int = 180  # seconds
+    MISFIRE_GRACE_TIME: int = 300  # seconds
```

Comment:
IMHO it's better to add units to option name

Reply:
you mean:
`SCHEDULER_TRANSFER_FETCHING_TIMEOUT: int = 180`?

Reply:
I mean `export SYNCMASTER__SCHEDULER__TRANSFER_FETCHING_TIMEOUT_SECONDS=180`. Timeout can be seconds, milliseconds, etc.